### PR TITLE
blog: Add Free tier API with Apache APISIX post

### PR DESCRIPTION
**Pull request by nfrankel**
Screenshot 2024-08-03 at 11 35 00

### CODE REVIEW

File: blog/en/blog/2024/08/01/free-tier-api-apisix.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+    title: Author
+    url: https://github.com/nfrankel
+    image_url: https://avatars.githubusercontent.com/u/752258
+keywords:
+  - APISIX
+  - Free tier
+  - Rate limiting
+  - Consumer
+description: >
+  Lots of service providers offer a free tier of their service. The idea is to let you kick their service's tires freely. If you need to go above the free tier at any point, you'll likely stay on the service and pay. In this day and age, most services are online and accessible via an API. Today, we will implement a free tier with Apache APISIX.
```
Comment (kayx23) [inline blog/en/blog/2024/08/01/free-tier-api-apisix.md:14]: > implement a free tier with Apache APISIX

I am a bit confused with this wording. Usually you would say developers use the free tier services to implement something. Are you writing from the perspective of a MSP?

**Review by pottekkat — APPROVED** [review]

File: blog/en/blog/2024/08/01/free-tier-api-apisix.md (hunk 1 above)
Comment (nfrankel) [reply blog/en/blog/2024/08/01/free-tier-api-apisix.md:14]: I am. The wording seems to me quite clear:

>Today, we will implement a free tier with Apache APISIX.

**Review by kayx23 — APPROVED** [review]

**Comment by kayx23** [comment]
Should we merge this?

**Comment by nfrankel** [comment]
@kayx23  Sorry, I didn't see that you approved.

**Pull request merged**